src/highlighter.js

```diff
@@ -1084,7 +1142,11 @@ export default function (options) {
      * @returns {boolean}
      */
     function isWrappingNode(node) {
-        return isElement(node) && node.tagName.toLowerCase() === 'span' && highlightingClasses.includes(node.className);
+        return (
+            isElement(node) &&
+            node.tagName.toLowerCase() === 'span' &&
+            highlightingClasses.includes(node.className?.split(' ')[0])
```

Comment:
Making `className` a concatenated string was the simplest. `className` and `a.className !== b.className` is used very often in this module, and I'm afraid to touch anything here. I'll see if I can do a more proper solution without a lot of refactoring - agree, it's weird now. 

It's not in the docs, but I'm quite sure that  browser will always return the same sequence in `className`,  because it's like class attribute string, which certainly doesn't change randomly. And `classList.add` is never supposed to be used on these nodes (at the moment). 

PS: `color` and not additional data-property is used as commentId, because:
- if color is the same, adjacent nodes will be merged together in single highlight, but we need to keep separate highlights, even if they are siblings
- to have less changes in this module